src/constants.rs

```diff
@@ -1,5 +1,5 @@
-// MinSQL
-// Copyright (C) 2019  MinIO
+// This file is part of the MinSQL
+// Copyright (c) 2019 MinIO, Inc.s
```

Comment:
looks like some typo here at the end of `MinIO, Inc.s` also perhaps ` This file is part of the MinSQL`  should be just "This file is part of MinSQL" 

I generally refer to AGPL projects written by GNU folks themselves one such example is https://git.gnunet.org/gnunet.git/tree/src/dns/dns.h?id=4c1ef8149081635778c52c0be9ca49aadaea137c 

Looks like the license-templates project is not entirely correct and misleading in some ways.

Reply:
i think we are aligned with the text listed on
https://www.gnu.org/licenses/agpl-3.0.en.html

Reply:
I also fixed the typo. I also remove `the` as the suggested license header says "this file is part of"